src/bootstrap/cc_detect.rs

```diff
@@ -37,7 +37,9 @@ use crate::{Build, GitRepo};
 // try to infer the archiver path from the C compiler path.
 // In the future this logic should be replaced by calling into the `cc` crate.
 fn cc2ar(cc: &Path, target: &str) -> Option<PathBuf> {
```

Comment:
I don't think the default that this falls back to (without checking for existence) is good. But I don't want to change it because it might break things.